CONTRIBUTING.md

```diff
@@ -8,9 +8,15 @@ For small bug-fixes, documentation updates, or other trivial changes, feel free
 
 If the changes are larger (API design, architecture, etc), [opening an issue](https://github.com/player-ui/player/issues/new/choose) can be helpful to reduce implementation churn as we hash out the design.
 
-## Building and Testing Locally
+## Requirements
+* [npm >= 8.19.2](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+* [yarn >= 1.22.19](https://yarnpkg.com/)
+* [Swift >= 5.2](https://www.swift.org/download/)
+* [Android NDK = 19.2.5345600](https://github.com/android/ndk/wiki/Unsupported-Downloads#r19c). You'll need to add `ANDROID_NDK_HOME` to your environment manually.
```

Comment:
We might need to explicitly call out the incompatibility with NDK version >21

Reply:
do we need bazel 6.x to bump up the android ndk?

Reply:
How's this?

> * [Android NDK >= 19.2.5345600, <= 21](https://github.com/android/ndk/wiki/Unsupported-Downloads#r19c). Any version > 22 will not work, period. You'll need to add `ANDROID_NDK_HOME` to your environment manually.

Reply:
I looked around, it looks like it's not supported yet.. there's [this](https://github.com/bazelbuild/bazel/pull/14077) PR but it's been up for a while and according to [this](https://github.com/bazelbuild/bazel/issues/12889) thread, that will no longer be the solution going forward, and there's supposed to be a `rules_android_ndk` that supports NDK r25. I don't know if that'll require bazel 6.x atm

Reply:
https://github.com/ahumesky/rules_android_ndk
this specifically says version 25, so i'm assuming 22, 23, 24 don't work lol